quiz/locale/de/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,476 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
```

Comment:
Are these files are necessary?

Reply:
I think they are not.also remove the comments in urls, and installed apps after you try in your system and doing some modifications

Reply:
Sure